client/src/components/job-lead-dashboard-component/job-lead-dashboard-filters/index.jsx

```diff
@@ -0,0 +1,463 @@
+                style={{
+                  borderWidth: "10px",
+                }}
+                InputProps={{
+                  endAdornment: (
+                    <InputAdornment position="end">
+                      <SearchIcon />
+                    </InputAdornment>
+                  ),
+                }}
+              />
+            </Typography>
+            {/* Date Created Filter */}
+            <Typography
+              sx={{ fontSize: 14 }}
+              color="text.secondary"
+              align="left"
+              gutterBottom
+            >
+              Date Created
+            </Typography>
+            <DatePicker
+              label="From"
+              value={startDateExpired}
+              onChange={setStartDateExpired}
+              renderInput={(params) => {
+                // eslint-disable-next-line react/jsx-props-no-spreading
+                return <TextField {...params} size="small" fullWidth />;
+              }}
+            />
+            <DatePicker
+              label="Until"
+              value={endDateExpired}
+              onChange={setEndDateExpired}
+              renderInput={(params) => {
+                // eslint-disable-next-line react/jsx-props-no-spreading
+                return <TextField {...params} size="small" fullWidth />;
+              }}
+            />
+            {/* Date Expired Filter */}
+            <Typography
+              sx={{ fontSize: 14 }}
+              color="text.secondary"
+              align="left"
+              gutterBottom
+            >
+              Expiry Date
+            </Typography>
+            <DatePicker
+              label="From"
+              value={startDateCreated}
+              onChange={setStartDateCreated}
+              renderInput={(params) => {
+                // eslint-disable-next-line react/jsx-props-no-spreading
+                return <TextField {...params} size="small" fullWidth />;
+              }}
+            />
+            <DatePicker
+              label="Until"
+              value={endDateCreated}
+              onChange={setEndDateCreated}
+              renderInput={(params) => {
+                // eslint-disable-next-line react/jsx-props-no-spreading
+                return <TextField {...params} size="small" fullWidth />;
+              }}
+            />
+            {/* Compensation Filter */}
+            <Typography
+              sx={{ fontSize: 14 }}
+              color="text.secondary"
+              align="left"
+              gutterBottom
+            >
+              Compensation
+            </Typography>
+            <Box sx={{ px: 2 }}>
+              <Slider
+                value={compensationRange}
+                onChange={onCompensationRangeChange}
+                valueLabelDisplay="auto"
+                getAriaValueText={valuetext}
+                disableSwap
+                step={1}
+                min={minCompensation}
+                max={maxCompensation}
```

Comment:
Both min and max should be set to a specified range. Maybe 0 to 100 for now and then we can make changes after some user feedback.